src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/AutoInsert/OnAutoInsertEndpointTest.cs

```diff
@@ -161,13 +192,10 @@ private class TestOnAutoInsertProvider(string triggerCharacter, bool canResolve)
 
         public string TriggerCharacter { get; } = triggerCharacter;
 
-        // Disabling because [NotNullWhen] is available in two Assemblies and causes warnings
-        public bool TryResolveInsertion(Position position, FormattingContext context, [NotNullWhen(true)] out TextEdit? edit, out InsertTextFormat format)
+        public VSInternalDocumentOnAutoInsertResponseItem? TryResolveInsertion(Position position, RazorCodeDocument codeDocument, bool enableAutoClosingTags)
         {
             Called = true;
-            edit = ResolvedTextEdit!;
-            format = default;
-            return canResolve;
+            return canResolve ? new() { TextEdit = ResolvedTextEdit!, TextEditFormat = default } : default;
```

Comment:
1. Please prefer `null` rather than `default`.
2. The default value of `VSInternalDocumentOnAutoInsertResponseItem.TextEditFormat` is `InsertTextFormat.PlainText`, which has a value of `1`. By specifying `default`, you're setting this to `0`. Is that intended?
3. I don't believe the `!` is necessary on `ResolvedTextEdit`.

Reply:
I changed the signature per earlier suggestion. # 2 and # 3 above were in original text, but you are right, "!" wasn't necessary, and I switched format to plain text.

Reply:
I see that `TextEdit` is non-nullable now. Thanks!